base_layer/wallet/src/storage/sqlite_utilities/mod.rs

```diff
@@ -142,3 +145,25 @@ pub fn initialize_sqlite_database_backends<P: AsRef<Path>>(
         key_manager_backend,
     ))
 }
+
+pub fn get_last_network_and_version<P: AsRef<Path>>(db_path: P) -> Result<(String, String), WalletStorageError> {
```

Comment:
Split this into `get_last_network`, and `get_last_version `and rather than return a "n/a"string, return option<String>
By returning a "n/a"string we need to do string processing in order to find out it does not have a stored string.